code/sarix_model/sarix_model.py

```diff
@@ -37,580 +39,82 @@
-             inplace=True)
-  dat['agg_level'] = np.where(dat['agg_level'] == 'National',
-                              'national',
-                              dat['agg_level'].str[:-1].str.lower())
-  dat = dat.sort_values(by=['season', 'season_week'])
-  
-  # for early seasons, drop out-of-season weeks with no reporting
-  early_seasons = [str(yyyy) + '/' + str(yyyy + 1)[2:] for yyyy in range(1997, 2002)]
-  early_in_season_weeks = [w for w in range(10, 43)]
-  first_report_season = ['2002/03']
-  first_report_in_season_weeks = [w for w in range(10, 53)]
-  dat = dat[
-    (dat.season.isin(early_seasons) & dat.season_week.isin(early_in_season_weeks)) |
-    (dat.season.isin(first_report_season) & dat.season_week.isin(first_report_in_season_weeks)) |
-    (~dat.season.isin(early_seasons + first_report_season))]
-  
-  # region 10 data prior to 2010/11 is bad, drop it
-  dat = dat[
-    ~((dat['location'] == 'Region 10') & (dat['season'] < '2010/11'))
-  ]
-  
-  if scale_to_positive:
-    dat = pd.merge(
-      left=dat,
-      right=load_who_nrevss_positive(),
-      how='left',
-      on=['agg_level', 'location', 'season', 'season_week'])
-    dat['inc'] = dat['inc'] * dat['percent_positive'] / 100.0
-    dat.drop('percent_positive', axis=1)
-
-  if drop_pandemic_seasons:
-    dat.loc[dat['season'].isin(['2008/09', '2009/10', '2020/21', '2021/22']), 'inc'] = np.nan
-
-  # if requested, make adjustments for overall season burden
-  # if burden_adj:
-  #   hosp_burden_adj = calc_hosp_burden_adj()
-  #   dat = pd.merge(dat, hosp_burden_adj, on='season')
-  #   dat['inc'] = dat['inc'] * dat['adj_factor']
-
-  dat = dat[['agg_level', 'location', 'season', 'season_week', 'wk_end_date', 'inc']]
-  dat['source'] = 'ilinet'
-  return dat
-
-
-
-def load_hhs(rates=True):
-  dat = pd.read_csv("../../data-raw/influenza-hhs/hhs.csv")
-  dat.rename(columns={'date': 'wk_end_date'}, inplace=True)
-
-  ew_str = dat.apply(date_to_ew_str, axis=1)
-  dat['season'] = convert_epiweek_to_season(ew_str)
-  dat['season_week'] = convert_epiweek_to_season_week(ew_str)
-  dat = dat.sort_values(by=['season', 'season_week'])
-  
-  if rates:
-    pops = load_us_census()
-    dat = dat.merge(pops, on = ['location', 'season'], how='left') \
-      .assign(inc=lambda x: x['inc'] / x['pop'] * 100000)
-
-  dat['wk_end_date'] = pd.to_datetime(dat['wk_end_date'])
-  
-  dat['agg_level'] = np.where(dat['location'] == 'US', 'national', 'state')
-  dat = dat[['agg_level', 'location', 'season', 'season_week', 'wk_end_date', 'inc']]
-  dat['source'] = 'hhs'
-  return dat
-
-
-
-def get_season_hol(start_year):
-  holiday_cal = USFederalHolidayCalendar()
-  hol = holiday_cal.holidays(
-    start=datetime.datetime(year=start_year, month=7, day=1),
-    end=datetime.datetime(year=start_year+1, month=6, day=1),
-    return_name=True)
+def get_sarix_preds(power_transform):
+    fdl = FluDataLoader('../../data-raw')
```

Comment:
This reminds me of a conversation that @rogersbw and I had a while back...in a Python project, where is the best place to put a data folder that contains files required by the app?

The question is more relevant if this experiment code is something you're planning to package, because ensuring the package includes the files is possible when a "data" folder exists outside of the code/src folder, but a bit more futzy.

Additionally, putting the "data" folder relative to the app makes it easier to get the location programmatically, in a way that will work cross-platform.

For example, assuming that this code someday becomes an installable Python app (i.e., with a pyproject.toml), you can get a reference to wherever the app is installed and find the "data" folder from there:

```python
import importlib
from pathlib import Path
# as a Path object
dir = Path(importlib.util.find_spec("name_of_module").origin).parent
# as a string
dir = str(Path(importlib.util.find_spec("name_of_module").origin).parent)
```

Reply:
My eventual plan on this front is to put all the files in an s3 bucket, since that will make them accessible to both R and python -- but I agree that this has been cumbersome.  Will keep this general idea in mind!